### PR TITLE
Stop packaging the test data

### LINKED ISSUE #42: Remove test files from distributed package
Currently the test FITS files are included in the MANIFEST, which means that the `pip install` size is 47MB.  It should be less than 1MB.  Also, the test files don't need to be 11MB in size.
```
rey> pip install drizzle --no-cache-dir
Collecting drizzle
  Downloading https://files.pythonhosted.org/packages/63/a8/d73561940986e75b3874e35d9353d1be5734d62daefbff912d53850bd98d/drizzle-1.12.tar.gz (47.0MB)
Requirement already satisfied: astropy in /Users/jdavies/miniconda3/envs/test/lib/python3.7/site-packages (from drizzle) (3.1.2)
Requirement already satisfied: numpy>=1.13.0 in /Users/jdavies/miniconda3/envs/test/lib/python3.7/site-packages (from astropy->drizzle) (1.16.2)
Installing collected packages: drizzle
  Running setup.py install for drizzle ... done
Successfully installed drizzle-1.12
```
Maybe move the tests out of the distributed package too.

### PR DESCRIPTION
**Pull request by jdavies-st**
Before
```
$ ls -lh dist
total 186480
-rw-rw-r--  1 jdavies  STSCI\science    45M Mar 29 15:08 drizzle-1.13.dev357-cp36-cp36m-macosx_10_7_x86_64.whl
-rw-rw-r--  1 jdavies  STSCI\science    44M Mar 29 15:08 drizzle-1.13.dev357.tar.gz
```
After
```
$ ls -lh dist
total 648
-rw-rw-r--  1 jdavies  STSCI\science   143K Mar 29 15:10 drizzle-1.13.dev358-cp36-cp36m-macosx_10_7_x86_64.whl
-rw-rw-r--  1 jdavies  STSCI\science   178K Mar 29 15:10 drizzle-1.13.dev358.tar.gz
```
Closes #44.

Fixes #42.

### CODE REVIEW

**Comment by pllim** [comment]
Is this a duplicate of #44?

**Comment by jdavies-st** [comment]
Yes.  It is my attempt to get the CI tests to pass on #44.  As you can see, not successful.  `astropy_helpers` does some, interesting, stuff that is not at all clear.  Lots of hidden state.

File: .travis.yml [hunk 1]
```diff
@@ -31,12 +31,12 @@ env:
...
         - PYTHON_VERSION=3.6
         - NUMPY_VERSION=stable
-        - MAIN_CMD='python setup.py'
         - CONDA_DEPENDENCIES='Cython'
         - CONDA_ALL_DEPENDENCIES='astropy Cython'
         - SETUP_XVFB=True
         - EVENT_TYPE='push pull_request'
-        - SETUP_CMD='test'
+        - INSTALL_CMD='python setup.py build_ext --inplace'
+        - TEST_CMD='pytest'
```
Comment (pllim) [inline .travis.yml]: I wouldn't recommend calling `pytest` directly. Use `setup.py test` as before. The latter invokes some custom test runner magic.

**Comment by pllim** [comment]
xref https://github.com/spacetelescope/drizzle/pull/44#issuecomment-478352683

**Pull request closed without merging**